docs/config.md

```diff
@@ -2082,6 +2082,16 @@ by providing it a channel of all the keys it is expected to contain according
 to the [`Provide.Strategy`](#providestrategy). During this operation,
 all keys in the `Keystore` are purged, and only the given ones remain scheduled.
 
+**Persistent reprovide cycle state:** When Provide Sweep is enabled, the
+reprovide cycle state is persisted to the datastore. On restart, Kubo
+automatically resumes from where it left off. If the node was offline for an
+extended period, all CIDs that haven't been reprovided within the configured
+[`Provide.DHT.Interval`](#providedhtinterval) are immediately queued for
+reproviding. Additionally, the provide queue is persisted on shutdown and
+restored on startup, ensuring no pending provide operations are lost. If you
+don't want to keep the persisted provider state from a previous run, you can
+start the daemon with the `--provider-fresh-start` flag.
```

Comment:
@guillaumemichel since this can be only enabled/disabled before one runs `ipfs daemon`, could we simplify and make it a config `Flag` (enabled by default)?

Adding parameter to `ipfs dameon`, while does the trick, is bit awkward UX, because manu users use docker or don't have access to CLI (e.g. in IPFS Desktop), while at the same time they do to the JSON config.

I'm thinking replacing `--provider-fresh-start` CLI flag with a `Flag` in JSON config at `Provide.DHT.ResumeEnabled` – thoughts?